packages/transformer/src/test/standalone/IModelTransformer.test.ts

```diff
@@ -282,7 +284,9 @@ describe("IModelTransformer", () => {
       targetDb.saveChanges();
       TransformerExtensiveTestScenario.assertTargetDbContents(
         sourceDb,
-        targetDb
+        targetDb,
+        undefined,
+        true
```

Comment:
having `undefined` in the middle of parameters is a sign that you should refactor to use an argument object with optional properties. There are only 4 calls to `assertTargetDbContents` so it shouldn't be that bad

Reply:
changed it